heroku/cli
# refactor(pg-v5): Move command pg:unfollow to oclif

### Description
https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001bDEprYAG/view

This converts `pg:unfollow` to oclif and updates its associated tests.

### Testing

1) Pull down this branch
2) Have an app ready with two databases. One database is a follower of the other
3) Run `./bin/run pg:unfollow --app APP DATABASE2`
4) Verify the command succeeds and DATABASE2 is no longer a follower
5) Compare with prod
